eng/CredScanSuppression.json

```diff
@@ -27,6 +27,10 @@
                 "NewPassword1!",
                 "Password*",
                 "123",
+                "http://foo:bar@1.1.1.1:3000",
+                "http://foo1:bar1@1.1.1.1:3000",
+                "new_secret",
```

Comment:
We should try to consolidate and reuse existing placeholders as much as possible can these be collapsed into an existing one?